api/SVGAElement.json

```
@@ -95,6 +95,54 @@
           }
         }
       },
+      "href": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/links.html#dom-hyperlink-href",
```

```suggestion
          "spec_url": "https://svgwg.org/svg2-draft/single-page.html#types-__svg__SVGURIReference__href",
```

The `href` property defined in HTML is a `DOMString`, while this is an `SVGAnimatedString`. See also https://github.com/w3c/svgwg/issues/312 for some weirdness with how this property is defined. There's no weirdness in implementation though, so we can add this in BCD no problem.